src/app/(default)/climb/[[...slug]]/components/ClimbData.tsx

```diff
@@ -28,7 +28,7 @@ export const ClimbData: React.FC<ClimbType & Pick<AreaType, 'gradeContext'> & {
           {gradeStr != null && (
             <RouteGradeChip gradeStr={gradeStr} safety={safety} />
           )}
-          <RouteTypeChips type={type} />
+          <RouteTypeChips types={getDisciplineList(type)} />
```

Comment:
For me, `<RouteTypeChips types={getDisciplineList(sanitizedDisciplines)} />` worked to remove the `__TYPENAME` from displaying (although it gave me warnings about not passing the correct data type to the getDisciplineList function).